core-bundle/src/Controller/ContentElement/PlayerController.php

```diff
@@ -95,9 +95,22 @@ private function buildVideoFigureData(ContentModel $model, array $sourceFiles):
             ->setIfExists('preload', $model->playerPreload)
         ;
 
-        $range = $model->playerStart || $model->playerStop
-            ? \sprintf('#t=%s', implode(',', [$model->playerStart ?: '', $model->playerStop ?: '']))
-            : '';
+        $range = '';
+
+        if ($model->playerStart || $model->playerStop)
+        {
+            $range = '#t=';
+
+            if ($model->playerStart)
+            {
+                $range .= $model->playerStart;
+            }
+
+            if ($model->playerStop)
+            {
+                $range .= ',' . $model->playerStop;
+            }
+        }
```

Comment:
Could be shortened with `array_filter()`:

```suggestion
        $range = $model->playerStart || $model->playerStop
            ? \sprintf('#t=%s', implode(',', array_filter([$model->playerStart ?: '', $model->playerStop ?: ''])))
            : '';
```

Reply:
@fritzmg I think this wouldn't work in situations where you just want to a player stop time with no start time given, e.g. `foo.mp4#t=,33`.

Reply:
Hm, never mind, this won't work properly if you only define a stop time.

Reply:
I have updated the suggestion accordingly.

Reply:
@fritzmg or maybe like this if you want it to set to 0 explicitely, whatever you prefer  👍🏻 
![image](https://github.com/user-attachments/assets/3d715bc6-c622-42c0-8a3a-7f04c963390d)